services/visual-studio-app-center/visual-studio-app-center-releases-version.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderVersionBadge } = require('../version')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+
+const schema = Joi.object({
+  version: Joi.string().required(),
+  short_version: Joi.string().required(),
+}).required()
+
+module.exports = class VisualStudioAppCenterReleasesVersion extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'visual-studio-app-center/releases/version',
+      pattern: ':owner/:app/:token',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Visual Studio App Center Releases',
+        namedParams: {
+          owner: 'jct',
+          app: 'my-amazing-app',
+          token: 'ac70cv...',
+        },
+        staticPreview: renderVersionBadge({ version: '1.0 (4)' }),
+        keywords,
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'release',
+    }
+  }
+
+  async handle({ owner, app, token }) {
+    const { version, short_version } = await this.fetch({
+      owner,
+      app,
+      token,
+      schema,
+    })
+    return renderVersionBadge({
+      version: `${short_version} (${version})`,
```

Comment:
Based on the example and some of the service tests, I'm not sure I follow the difference between these two fields, as it seems like `short_version` is actually longer than `version`?

Should we really display both of them together like this, or perhaps provide a param/option to the user to allow them to specify which one they'd prefer? Perhaps a query param like `?versionFormat=short`

Reply:
`short_version` is the actual application version, whereas `version` is the release number.

Although I'm not personally sure how useful the release number would be on it's own.

Reply:
Ah okay. So just weird/somewhat misleading field names in the API response. 

Is that the common format users see in App Center? If so then agreed this makes the most sense

Reply:
Yeah, that is the format displayed in App Center right now.

Reply:
Thanks!